module/x/gravity/keeper/keeper.go

```diff
@@ -726,3 +726,10 @@ func (k Keeper) MigrateGravityContract(ctx sdk.Context, newBridgeAddress string,
 	params.BridgeEthereumAddress = newBridgeAddress
 	k.SetParams(ctx, params)
 }
+
+// disableBridge disable the bridge processing all outgoing and ingoing transactions
+func (k Keeper) disableBridge(ctx sdk.Context) {
+	gravityParam := k.GetParams(ctx)
```

Comment:
adding a log message may be useful if this can be triggered from different places?

Reply:
added a log.

I think it will be easy to detect since no more transaction will be processed